import/xsl/dspace-xoai.xsl

```diff
@@ -0,0 +1,164 @@
+<!-- 
+    The contents of this file are subject to the license and copyright
+    detailed in the LICENSE and NOTICE files at the root of the source
+    tree and available online at
+    http://www.dspace.org/license/
+	Developed by DSpace @ Lyncode <dspace@lyncode.com>
+ --><xsl:stylesheet xmlns:dcterms="http://purl.org/dc/terms/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="1.0" xmlns:php="http://php.net/xsl" xsi:schemaLocation="http://www.w3.org/2005/Atom http://www.kbcafe.com/rss/atom.xsd.xml">
+   <xsl:output method="xml" indent="yes" encoding="utf-8"/>
+   <xsl:param name="collection">DSpace</xsl:param>
+   <xsl:param name="urlPrefix">http</xsl:param>
+   <xsl:param name="geographic">false</xsl:param>
+   <xsl:param name="id_tag_name">identifier</xsl:param>
+   <xsl:param name="change_tracking_core">biblio</xsl:param>
+   <xsl:param name="change_tracking_date_tag_name"></xsl:param>
+   <xsl:param name="workKey_include_regEx"/>
+   <xsl:param name="workKey_exclude_regEx"/>
+   <xsl:param name="workKey_transliterator_rules">:: NFD; :: lower; :: Latin; :: [^[:letter:] [:number:]] Remove; :: NFKC;</xsl:param>
+   <xsl:template match="/">
+      <add>
+         <doc>
+	    <!-- RECORD ID -->
+	    <field name="id">
+               <xsl:value-of select="//*[name()=$id_tag_name]"/>
+            </field>
+            <!-- RECORD FORMAT -->
+	    <field name="record_format">dspace</field>
+            <!-- INSTITUTION -->
+	    <field name="institution">
+		<xsl:if test="//*[@name='publisher']">
+	            <xsl:value-of select="//*[@name='publisher']/*/*[@name='value']"/>
```

Comment:
Usually institution refers to the institution hosting DSpace -- in our other examples it's hard-coded to use a property. Is there a reason why you're putting the publisher here?

Reply:
thanks @demiankatz that was a mistake on my part. I've fixed it up in my followup.